main.tf

```diff
@@ -44,6 +44,7 @@ module "cloud_sql" {
 
   enable_private_ip  = var.enable_cloud_sql_private_ip
   private_network_id = var.private_network_id
+  postgres_verion    = var.sql_pg_version
```

Comment:
@zbpvarun-tamr we have to add `postgres_version` in the `terraform-gcp-tamr-cloud-sql` module as well.

Reply:
@zbpvarun-tamr you have to upgrade the module version to `v6.1.0`(The newer version)

Reply:
Thanks @karankaran1908. Just pushed another commit.